macos-deployment/make-app.sh

```diff
@@ -26,9 +26,11 @@ export LC_ALL=en_US.UTF-8
 export LANG=en_US.UTF-8
 export LANGUAGE=en_US.UTF-8
 
+CONFIG_FILE="\$HOME/geoweaver/.geoweaver_config"
```

Comment:
What if the geoweaver folder doesn't exist? You should create the folder if it doesn't exist.

Reply:
It is being created at the line number 190